ynh_add_swap/ynh_add_swap

```diff
@@ -63,3 +71,17 @@ ynh_del_swap () {
 		rm /swap_$app
 	fi
 }
+
+# Check if the device of the main mountpoint "/" is an SD card
+#
+# return 1 if it's an SD card, else 0
+_ynh_is_main_device_is_sd_card () {
```

Comment:
We don't usually start function with an underscore, but put an 'internal' flag to exclude it from the documentation (I guess it was the idea).